sentry-core/build.gradle.kts

```
@@ -65,6 +66,18 @@ tasks {
     }
 }
 
+buildConfig {
+    useJavaOutput()
+    packageName("io.sentry.core")
```

It's package as in package-manager:

```suggestion
    packageName("maven:io.sentry.sentry-core")
```

---

Would change when we rename to `io.sentry`

---

This is a package where the plugin generates a class.